packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,20 @@
+{
+  "hosting_multisite_domain_configuration_home": "Dossier racine",
+  "hosting_multisite_domain_configuration_myfolder": "mondossier",
```

Comment:
Intentional ?

Reply:
I've just extracted the translations so component can have its own. No content update made